src/Google/Client.php

```diff
@@ -456,6 +456,8 @@ public function isAccessTokenExpired()
   {
     if (!$this->token) {
       return true;
+    } elseif (!isset($this->token['expires_in'])) {
```

Comment:
elseif not required since the previous statement returns true. Just add another if.